source/scenes/worldMap.js

```diff
@@ -16,23 +16,36 @@ export class WorldMap extends Phaser.Scene{
         this.load.image('world-map', 'source/assets/WorldMap.png');
     }
     create(){
+    	let mapCenterX = -1000;
+    	let mapCenterY = -600;
+    	const mapWidth = 2161;
+    	const mapHeight = 1166;
+    	const screenWidth = 765;
+    	const screenHeight = 503;
+
         // Background
-        let map = this.add.image(0, 0, 'world-map').setOrigin(0,0).setDepth(0).setInteractive();
-        this.input.setDraggable(map);
-        this.input.on('drag', function (pointer, gameObject, dragX, dragY) {
-            gameObject.x = dragX;
-            gameObject.y = dragY;
-        });
+        let map = this.add.image(0, 0, 'world-map').setOrigin(0,0).setDepth(0);
 
-        // City text
-        let tutorialIsland = this.add.text(600, 360, 'Tutorial Island', {fill: 'white', fontSize: '20px'}).setDepth(1);
+        // Color links if they haven't been unlocked yet
+        let fillColor = 'yellow';
+        if (!this.characterData.TUTORIAL_ISLAND.questCompleted) {
+            fillColor = 'white';
+        }
+
+        // Tutorial Island
+        const tutorialIslandX = 1100;
+        const tutorialIslandY = 1070;
+        let tutorialIsland = this.add.text(tutorialIslandX, tutorialIslandY, 'Tutorial Island', {fill: 'yellow', fontSize: '20px', style: 'bold'}).setDepth(1);
```

Comment:
Should extract out the constants such as lumvridge and the text property object

Reply:
Moved into constants.js under a separate export

Reply:
I think you should move it back in less files the better